src/seedu/addressbook/AddressBook.java

```diff
@@ -210,13 +210,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        communicateWithUser();
```

Comment:
i appreciate the thought process here, but again, try using a better name for this method. `communicateWithUser` doesn't give a sense of what the method is doing, no?